src/components/SecondaryPanes/BreakpointsContextMenu.js

```diff
@@ -187,37 +187,37 @@ export default function showContextMenu(props) {
   const hideDisableSelf = breakpoint.disabled;
 
   const items = [
-    { item: enableSelf, hidden: () => hideEnableSelf },
-    { item: enableAll, hidden: () => hideEnableAll },
-    { item: enableOthers, hidden: () => hideEnableOthers },
+    { item: enableSelfItem, hidden: () => hideEnableSelf },
```

Comment:
You might also rename the `hideEnableSelf` to `hideEnableSelfItem` here and in line 182. Same for the other `hide...`. this will make it crystal-clear what is meant.